mgmt-lambda/app.ts

```diff
@@ -16,7 +16,7 @@ export async function handler(event: APIGatewayProxyEventV2WithRequestContext<AP
     const authSettings = await getAuthSettings(secretManagerClient)
     const authToken = retrieveAuthToken(event)
     if (authToken !== authSettings.token) {
```

Comment:
If both tokens are empty, this is true, right? We should throw errors if these tokens are empty